### DISCUSSION TITLE
软件兼容问题

### DISCUSSION THREAD

**Discussion by splrad** [File-New-Project/EarTrumpet#830] [General]
Not compatible with StartlsBack++.
When StartlsBack++ is turned on and EarTrumpet is turned on, the software interface will be slow and flickering after clicking the software volume icon. Everything is normal after turning off StartlsBack++.
![动画](https://user-images.githubusercontent.com/44151430/118182262-5e358c00-b46b-11eb-9f98-ae52197f79d0.gif)

**Comment by Geeyun-JY3** [comment]
建议用英文。

还有你这个 discussion 类别选「Completed discussions」是干啥
